python/mdvtools/auth/auth0_provider.py

```
@@ -1,13 +1,16 @@
 import time
 import requests
 from authlib.integrations.flask_client import OAuth
-from flask import session, redirect, url_for
-from typing import Optional
+from flask import jsonify, session, redirect
+from typing import Optional, Tuple, Union
```

Ruff (0.11.9)

`typing.Tuple` imported but unused

Remove unused import

(F401)